deploy/contents/sync/scrap/get_services.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+get_services() {
+    local ssh_key="$1" # SSH key
+    local user="$2" # USER
+    local server_address="$3" # Server address
+    local output_dir="$4" #Output directory name where preferences will be saved
+    local output_file="cp-services" #Output file name where services will be saved
+
+    #SSH connection is done within this command
+    ssh -i $ssh_key -oStrictHostKeyChecking=no $user@$server_address sudo kubectl get nodes -o json | jq -r '.items[] | .metadata.labels | to_entries[] | select(.key | startswith("cloud-pipeline/")) | .key | sub("cloud-pipeline/";"") | select(startswith("cp-"))' > $output_dir/$output_file
```

Comment:
let's split this operation:
 - First, get the output from ssh
 - Second, check that command was successful
 - Finally, do the post processing of the data